lib/content-types/merge.js

```diff
@@ -101,6 +89,31 @@ const squish = (types) => {
           plugin.required = requiredLevel(attribute.required);
         }
 
+        // Organizes repeatable attributes
```

Comment:
This is _from_ @ayush2k's repeatables (mentioned this in comment to re-read this PR)
https://github.com/punchcard-cms/content-types/pull/53/files#diff-7632fb88b259681879331dad06f8f231R104

I had to grab this to move forward.